homeassistant/components/barry/sensor.py

```diff
@@ -0,0 +1,63 @@
+"""Platform for sensor integration."""
+from datetime import timedelta
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(hours=1)
+MIN_TIME_BETWEEN_UPDATES = timedelta(hours=1)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Barry sensor."""
+    barry_connection = hass.data.get(DOMAIN)
+    price_code = hass.data.get("price_code")
```

Comment:
This is set in the config entry, thus exists.

```suggestion
    price_code = hass.data["price_code"]
```